edb/schema/casts.py

```diff
@@ -245,6 +245,18 @@ def _classname_from_ast(
 
         return get_cast_fullname(schema, 'std', from_type, to_type)
 
+    def canonicalize_attributes(
+        self,
+        schema: s_schema.Schema,
+        context: sd.CommandContext,
+    ) -> s_schema.Schema:
```

Comment:
Add a docstring? This looks like a new method added to a few places

Reply:
It's not a new method, just an overload of an existing one.  I'll make sure there is a comment on the main definition, but I don't think there is a point in copy-pasting the docstring into every overload.